fe/fe-core/src/main/java/org/apache/doris/analysis/CreateStorageVaultStmt.java

```diff
@@ -102,6 +108,7 @@ public void analyze(Analyzer analyzer) throws UserException {
         if (type == null) {
             throw new AnalysisException("Storage Vault type can't be null");
         }
+        setAsDefault = Boolean.parseBoolean(properties.getOrDefault(SET_AS_DEFAULT, "false"));
```

Comment:
what if an arbitrary string passed in? e.g
properties (
"set_as_default" = "10086"
)

Reply:
It would throw one exception to user